app/services/data_migrations/fix_lat_long_flip_flops.rb

```diff
@@ -60,7 +60,7 @@ def change
 
     def providers
       if dry_run?
-        Provider.where(sync_courses: true).limit(50)
+        Provider.includes(:courses).where(courses: { id: nil }).limit(50)
```

Comment:
similar to above

Reply:
same should be `where.not`

Reply:
since this is reused now in 3 places maybe we can move it to the provider model under a method `with_courses` or something similar?